_drafts/2020-12-14-data_science_interview_prep_data_manipulation.md

```diff
@@ -0,0 +1,370 @@
+---
+layout: post
+title: "Data Science Interview Practice: Data Manipulation"
+description: >
+image: /files/switrs-dataset/tram_auto_crash_in_1957_frederiksplein_amsterdam.jpg
+show_lead_image: True
+image_alt: >
+categories:
+  - career-advice
+  - interview-prep
+---
+
+<!-- Simple script to call from the "Show/Hide" buttons-->
+<script type="text/javascript">
+function showhide(d) {
+  d.style.display = (d.style.display !== "none") ? "none" : "block";
```

Comment:
Prefer `elt` to `d`. Ternary operator unnecessarily trades legibility for brevity.